packages/fxa-settings/src/components/Settings/FlowRecoveryKeyDownload/index.tsx

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import FlowContainer from '../FlowContainer';
+import ProgressBar from '../ProgressBar';
+import DataBlock from '../../DataBlock';
+import {
+  FolderIconListItem,
+  GlobeIconListItem,
+  LockIconListItem,
+  PrinterIconListItem,
+} from '../../IconListItem';
+import ButtonDownloadRecoveryKey from '../../ButtonDownloadRecoveryKey';
+import { Link } from '@reach/router';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { RecoveryKeyImage } from '../../images';
+import { logViewEvent } from '../../../lib/metrics';
+
+export type FlowRecoveryKeyDownloadProps = {
+  localizedBackButtonTitle: string;
+  localizedPageTitle: string;
+  navigateBackward: () => void;
+  navigateForward: () => void;
+  recoveryKeyValue: string;
+  viewName: string;
+};
+
+export const FlowRecoveryKeyDownload = ({
+  localizedBackButtonTitle,
+  localizedPageTitle,
+  navigateBackward,
+  navigateForward,
+  recoveryKeyValue,
+  viewName,
+}: FlowRecoveryKeyDownloadProps) => {
+  return (
+    <FlowContainer
+      title={localizedPageTitle}
+      onBackButtonClick={navigateBackward}
+      {...{ localizedBackButtonTitle }}
+    >
+      <div className="w-full flex flex-col gap-4">
+        <ProgressBar currentStep={3} numberOfSteps={4} />
+        <RecoveryKeyImage className="my-6 mx-auto" />
+
+        <FtlMsg id="flow-recovery-key-download-heading">
+          <h2 className="font-bold text-xl">
+            {/* This is an em dash - add space? */}
+            Account recovery key generated — store it in a place you’ll remember
+          </h2>
+        </FtlMsg>
+        <FtlMsg id="flow-recovery-key-download-info">
+          <p className="text-sm">
+            This key will help recover your data if you forget your password.{' '}
+          </p>
+        </FtlMsg>
+        {/* TODO Add copy metric flow.settings.account-recovery.copy-option */}
+        <DataBlock value={recoveryKeyValue} isInline />
+        <FtlMsg id="flow-recovery-key-download-storage-ideas-heading">
+          <h3 className="font-bold -mb-4">
+            Some ideas for storing your recovery key
+          </h3>
+        </FtlMsg>
+        <ul>
+          <FolderIconListItem>
+            <FtlMsg id="flow-recovery-key-download-storage-ideas-folder">
+              <p>Memorable folder in your device</p>
+            </FtlMsg>
```

Comment:
Couldn't find this ID, but I did find a string match:
```ini
flow-recovery-key-download-storage-idea-folder = Memorable folder in your device
```

Looks like a pluralization issue on "idea":
```
flow-recovery-key-download-storage-idea-folder
flow-recovery-key-download-storage-ideas-folder
```